samples/nextjs/src/components/Layout.tsx

```diff
@@ -10,11 +10,14 @@ const LOGO_SIZE = { WIDTH: 221, HEIGHT: 48 };
 // Most apps may also wish to use GraphQL for their navigation construction; this sample does not simply to support disconnected mode.
 const Navigation = () => {
   const { t } = useI18n();
+  // The publicUrl will only be set for Editing Host builds.
+  // This is currently the only way to make Next.js disable prefetching for these completely (since this will be different than the window.location)
+  const publicUrl = process.env.publicUrl || '';
```

Comment:
@ambrauer Did you check another link's?
For example: `samples\nextjs\src\components\Styleguide-Multilingual\index.tsx` what about these links?

Reply:
@ambrauer Same for `samples\nextjs\src\components\Styleguide-CustomRouteType\index.tsx`. Now I have errors

Reply:
@ambrauer And `samples\nextjs\src\components\Styleguide-RouteFields\index.tsx`

Reply:
@ambrauer Seems like you need to extract this `getter` for `publicUrl` and reuse it

Reply:
@sc-illiakovalenko Ugh, yeah this is more invasive than I realized... The problem is that there's no easy way to universally disable prefetching for `next/link` (which we need to do for EE since it'll always attempt to retrieve prefetch data from current host). Let's discuss.